src/config/constants.js

```diff
@@ -0,0 +1,3 @@
+export default {
```

Comment:
Could you rewrite this as
`export const SOCIAL_ID = 'something'` 
That way you get import validation with eslint